scripts/terminology_valueset_sync.py

```diff
@@ -197,6 +197,24 @@ def get_all_loinc_terms_per_code(loinc_result: dict, loinc_order_rows) -> dict:
     #  from as well.  ie. TermDescriptions [], FormalName, and DisplayName.
     #  Will leave these out for now.
 
+    # Adding additional fields to extract terms from to help supplement
+    # data for learning in our models
+    # NOTE: We can change/remove these additional fields later or even
+    #  make them configurable
+
+    # More human centered name for the concept
+    if loinc_result.get("DisplayName") is not None:
+        result_row = {"code": result_code, "text": loinc_result.get("DisplayName")}
+        loinc_order_rows.append(result_row)
+    # Paragraph of information concerning the concept/code/term in question
+    if loinc_result.get("DefinitionDescription") is not None:
+        result_row = {"code": result_code, "text": loinc_result.get("DefinitionDescription")}
+        loinc_order_rows.append(result_row)
+    # ';' separated list of related terms to the concept/code/term in question
+    if loinc_result.get("RELATEDNAMES2") is not None:
+        result_row = {"code": result_code, "text": loinc_result.get("RELATEDNAMES2")}
+        loinc_order_rows.append(result_row)
+
```

Comment:
Is it likely that a single `loinc_result` would have more than one of a `DisplayName`, `DefinitionDescription`, and `RELATEDNAMES2`? Do we want one row per `DisplayName`, `DefinitionDescription`, and `RELATEDNAMES2` or one row per `loinc_result`? Just trying to understand if we need multiple if statements and how we want the output to look

Reply:
Does that answer your questions?  Happy to discuss further!  @m-goggins

Reply:
Got it.  From my experience that wouldn't occur.  The various 'name/description' elements, data fields, are different and there for a reason.  However, I did find some other oddities in some of the descriptions that are not relevant to the actual code, and those we 'should' filter.  (see our [slack conversation](https://skylight-hq.slack.com/archives/C095WTSJR29/p1757003460019569?thread_ts=1757000965.769919&cid=C095WTSJR29) - topic 1. in the thread from Brandon and myself.  [Examples of data as well - topic 1. again] (https://skylight-hq.slack.com/archives/C095WTSJR29/p1757005120221369?thread_ts=1757000965.769919&cid=C095WTSJR29)).